### ISSUE TITLE
Proposal to migrate from CoffeeScript to...

### ISSUE THREAD

**Issue by geoffmiller** [chutterbox/chutter#13]
My vote would be something ES6 based like Babel.js or TypeScript. ES5/6 may make this project more accessible.

**Comment by ambewas** [comment]
I agree. Not knowing CS syntax could be a barrier.

**Comment by PritishC** [comment]
What about vanilla JS/ES5? Wouldn't ES6/TS have the same effect that CS is having right now?

**Comment by chutterbox** [comment]
@PritishC i agree with that entirely. vanilla js/es5 would make the most sense

**Comment by geoffmiller** [comment]
ES5 is fine with me. It will eliminate a build step and hopefully foster more contributions. 
Eventually I will get you all to ES6.

**Comment by chutterbox** [comment]
@geniusme That sounds good to me. I'll make an issue for it.
